src/main/fc/tasks.c

```diff
@@ -568,5 +575,10 @@ void tasksInit(void)
     setTaskEnabled(TASK_TELEMETRY_SBUS2, useSBUS2);
 #endif
 
+#ifdef USE_SBUS_OUTPUT
+    rescheduleTask(TASK_SBUS_OUTPUT, TASK_PERIOD_HZ(sbusOutConfig()->sbusRate));
+    setTaskEnabled(TASK_SBUS_OUTPUT, true);
```

Comment:
It appears the SBUS task is always enabled?
This is not good. Please enable it only if the output is configured.

Reply:
Added sbusOutIsEnabled() to check